sdk/keyvault/keyvault-certificates/src/index.ts

```diff
@@ -369,7 +376,7 @@ function toPublicPolicy(policy: CoreCertificatePolicy = {}): CertificatePolicy {
 }
 
 function toPublicIssuer(issuer: IssuerBundle = {}): CertificateIssuer {
-  const parsedId = parseKeyvaultEntityIdentifier("certificates", issuer.id);
+  const parsedId = new KeyVaultCertificatesIdentifier(issuer.id!);
```

Comment:
Why for issuer? We had said we weren't planning on this for issuers or other resources for now (no ask). It also wouldn't be in the right collection that you're effectively asserting above.

Reply:
Gotcha. I'll clean this up!